server/controllers/transactions.js

```diff
@@ -65,7 +66,7 @@ module.exports = function(app) {
       addTransactionToUser: ['createTransaction', (cb, results) => {
         var transaction = results.createTransaction;
 
-        if (user && user.addTransaction) {
+        if (user) {
           user
```

Comment:
If user is null, this will fail (that's why the `&& user.addTransaction` was there)